docs/ThemeColors.stories.tsx

```diff
@@ -103,6 +108,7 @@ export const CSSDarkTheme = {
                 key={name}
                 color={color}
                 name={colorName}
+                backgroundColor={colorName}
```

Comment:
And here
<img width="760" alt="Screenshot 2024-05-22 at 12 08 16 PM" src="https://github.com/MetaMask/design-tokens/assets/8112138/dbfa65a4-0007-4dfc-8cef-76826a031e1c">
<img width="757" alt="Screenshot 2024-05-22 at 12 08 52 PM" src="https://github.com/MetaMask/design-tokens/assets/8112138/4481a5d1-d4af-4bc1-bf52-6aed12f46621">